webpack.common.js

```diff
@@ -0,0 +1,122 @@
+const path = require('path');
```

Comment:
I don't think it's a good idea to duplicate most parts of configs. They differ not so much and it makes it hard to understand what's going on. Plus there's just to much duplication between them now. I would put all config in just one file and tweak based on `mode` passed. @smelnikov what do you think?